.github/workflows/test.yml

```diff
@@ -0,0 +1,17 @@
+name: "build-test"
+on: # rebuild any PRs and main branch changes
+  pull_request:
```

Comment:
Limit the `pull_request` trigger as well to `master` and `releases/*`?

Reply:
changed